build/images/katas/Dockerfile

```diff
@@ -0,0 +1,67 @@
+# We use the iqsharp-base image, as that includes
```

Comment:
Could we be any chance move this file to `scripts` folder? As is, this change would create a `build` folder with exactly one script, and it makes sense to have all scripts in the same place.

Reply:
Since this is technically not a script, I though it would be better to have under a generic build folder, under images.
I think we should then move the scripts folder under the build folder (as it is with the other repos), but I didn't have time to do it yet (as I would need to update the QDK build)